scripts/check_zuul_files.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+files=("zuul.d/molecule.yaml" "zuul.d/projects.yaml")
+files_with_diff=()
+for file in "${files[@]}"; do
+    if ! git diff --exit-code --quiet -- "$file" > /dev/null 2>&1; then
+        files_with_diff+=("$file")
+        echo "Changes detected in $file"
```

Comment:
May add some more details in the print


        echo "Manual changes are not allowed in this file."
        echo "This file is autogenerated using 'make role_molecule'."

Reply:
Added. Thx Sandeep.